src/scripting/lua_game.cc

```diff
@@ -752,6 +753,31 @@ int LuaPlayer::mark_training_wheel_as_solved(lua_State* L) {
 	return 0;
 }
 
+/* RST
+   .. method:: run_training_wheel(name[, force])
+
+      Trigger running a training wheel while skipping the dependency check.
+      Previously solved training wheels will be ignored if ``force == true``.
```

Comment:
Add a note that the wheel's dependency requirements are ignored regardless of whether `force` is `true` or `false`